yggnode.py

```diff
@@ -16,119 +16,126 @@
+        subID = cfg["sub-Categories"]["id"]
+        subNAME = cfg["sub-Categories"]["idLabel"]
+        catdata = dict()
+        subdata = dict()
+        for index in range(len(catID)):
+            key = str(catNAME[index])
+            url = f"{server}/rss?id={str(catID[index])}&passkey={passkey}"
+            catdata[key] = url
+        for index in range(len(subID)):
+            key = str(subNAME[index])
+            url = f"{server}/rss?id={str(subID[index])}&passkey={passkey}"
+            subdata[key] = url
+        return render_template('genlinks.html', catdata=catdata, subdata=subdata, server=server, name=name)
+
+# Status route
+
 
 @app.route('/status', methods=['GET'])
 def getStatus():
-    confFile = open(os.getcwd() + '/annexes.yml', 'r')
-    serverConfiguration = yaml.safe_load(confFile)
-    confFile.close()
-    now = time.time()
-    renderTxt = "<head><title>last resync</title></head>Time since last synchronisation:<br><br>"
-    for index in range(len(serverConfiguration["Categories"]["id"])):
-        if not os.path.exists (os.getcwd() + "/rss/" + str(serverConfiguration["Categories"]["id"][index]) + ".xml"):
-            renderTxt += "<strong>" + serverConfiguration["Categories"]["idLabel"][index] + "</strong> : never synchronized<br>"
-        else:
-            renderTxt += "<strong>" + serverConfiguration["Categories"]["idLabel"][index] + "</strong> : " + str(timeUnderstandable(now - os.stat(os.getcwd() + "/rss/" + str(serverConfiguration["Categories"]["id"][index]) + ".xml").st_mtime)) + "<br>"
-    renderTxt += "<br><br>"
-    for index in range(len(serverConfiguration["sub-Categories"]["id"])):
-        if not os.path.exists (os.getcwd() + "/rss/" + str(serverConfiguration["sub-Categories"]["id"][index]) + ".xml"):
-            renderTxt += "<strong>" + serverConfiguration["sub-Categories"]["idLabel"][index] + "</strong> : never synchronized<br>"
-        else :
-            renderTxt += "<strong>" + serverConfiguration["sub-Categories"]["idLabel"][index] + "</strong> : " + str(timeUnderstandable(now - os.stat(os.getcwd() + "/rss/" + str(serverConfiguration["sub-Categories"]["id"][index]) + ".xml").st_mtime)) + "<br>"
-    return renderTxt
-
-def timeUnderstandable(sec):
-    intSeconds = int(sec)
-    hour = intSeconds // 3600
-    min = (intSeconds - hour*3600)//60
-    sec = intSeconds - hour*3600 - min * 60
-    return "%02dH %02dm %02ds" % (hour, min, sec)
+    with open("config/annexes.yml", 'r') as ymlfile:
+        cfg = yaml.load(ymlfile, Loader=yaml.FullLoader)
+    humanize.i18n.activate("fr_FR")
+    server = f'{str(cfg["node"]["protocol"])}://{str(cfg["node"]["ipAdress"])}:{str(cfg["node"]["port"])}'
+    name = f'{str(cfg["node"]["ipAdress"])}'
+    dataCat = dict()
+    dataSub = dict()   
+    for index in range(len(cfg["Categories"]["id"])):
+        now = time.time()
+        osModTime = os.stat(f'rss/{str(cfg["Categories"]["id"][index])}.xml').st_mtime
+        lastModTime = now - osModTime
```

Comment:
même combat ici.

Reply:
La ok sur le 'rss/{str(cfg["Categories"]["id"][index])}.xml'
Faut mettre un if sur les deux block for du coup

Reply:
Done